.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
+  TargetRubyVersion: 2.1
```

Comment:
Travis is testing an implemented version of Ruby.  This is the spec for the language spec. They aren't the same thing. We aren't using any language features of the 2.2+ ruby language spec. This calls that out.